apps/scan/frontend/Makefile

```diff
@@ -8,6 +8,7 @@ build: FORCE
 	pnpm install && \
 		pnpm --dir ../../../libs/ballot-interpreter build && \
 		pnpm --dir ../../../libs/pdi-scanner build && \
+		pnpm --dir ../backend build && \
```

Comment:
I am still a bit confused as to why this isn't getting run already. For `libs/ballot-interpreter` and `libs/pdi-scanner`, I understand that they're not referenced by source code as typical dependencies. We just use the Rust binaries that those libs build. So makes some sense that they get missed. For the `apps/scan/backend`, seems like we're definitely building it but maybe not using its `pnpm build` 🤔

Reply:
yeah we're definitely building it just...differently somehow lol

Reply:
I'm only 98% sure this fix will apply to prod but if I make images in the next day or two for the screen recorder I can double check this as well